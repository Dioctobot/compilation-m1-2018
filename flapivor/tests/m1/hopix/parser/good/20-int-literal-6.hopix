val x = 0o12345670