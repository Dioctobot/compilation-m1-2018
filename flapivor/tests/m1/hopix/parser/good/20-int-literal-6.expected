val x = 2739128
